pkg/build/graph/helpers.go

```diff
@@ -76,3 +78,27 @@ func defaultNamespace(value, defaultValue string) string {
 	}
 	return value
 }
+
+// BuildPointsToTag returns the buildConfig that points to the provided imageStreamTag.
+func BuildPointsToTag(g osgraph.Graph, istag graph.Node) (*buildgraph.BuildConfigNode, bool) {
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(istag, BuildOutputEdgeKind) {
+		return bcNode.(*buildgraph.BuildConfigNode), true
```

Comment:
> The name of the method makes me think I'd be accepting a BC/BCNode and IST/ISTNode and returning true if the relationship exists in the graph?

That's already happening with the boolean returned. I guess we could just return the bc and callers should check if the bc is nil or not (essentially the boolean abstracts away that check). A build config will always point to at most one istag, that's why we return the first in the list. I don't really care about the names at this point. Do you have any suggestion? We will revisit these helpers anyway once we make the graph api official.